me.elliotchance.MultiSelectorDynamicValue/MultiSelectorDynamicValue.js

```diff
@@ -16,17 +16,25 @@ var MultiSelector = function() {
   }
 
   this.title = function() {
-    return constructStringFromChoices(this.choices, this.separator);
+      if (!this.choices.length) {
+          return "Multi Selector \uD83D\uDCCB"
+      }
+
+      return "\uD83D\uDCCB"
+  }
+
+  this.text = function() {
+    return constructStringFromChoices(this.choices, this.separator) || null;
   }
 }
 
 MultiSelector.inputs = [
-    InputField("choices", "Choices", "KeyValueList"),
+    InputField("choices", "Choices", "KeyValueList", {persisted: true}),
```

Comment:
Just for clarification, the `persisted` flag does which one?

1. Remembers the value in the application data so that the next MultiSelectorDynamicValue uses the last used values by default?
2. Stores the values in the project file (instead of the application data) so that other people that open the project file will be able to have these values?